python/test/gluon/test_frontend.py

```diff
@@ -1661,46 +1661,23 @@ def test_amd_mfma_layout(target):
 
 
 @gluon.jit
-def add_int(a, b):
-    return a + b
+def amd_wmma_layout_kernel():
+    ttgl.full([64, 64], 0, ttgl.float16, layout=amd_layouts.AMDWMMALayout(version=2, transposed=True,
+                                                                          warps_per_cta=[1, 4]))
 
 
-@gluon.jit
```

Comment:
This test should already be covered in test_lowering.py::test_reduce_layouts. Removed here.